recipes/mqc/run_test.pl

```diff
@@ -0,0 +1,11 @@
+# Test file for testing Perl modules installation
+$|=1;
```

Comment:
Now I put the perl packages in the test section of the meta.yaml together with the python packages but he tries to test the perl packages with a python 'import' test, which makes the tests and thus also the whole build fail